skll/config.py

```diff
@@ -819,22 +811,18 @@ def _parse_config_file(config_path, log_level=logging.INFO):
         raise ValueError('The models path should not be set when task is '
                          '{}.'.format(task))
     if task == 'learning_curve':
-        if len(grid_objectives) > 0 and len(output_metrics) == 0:
-            logger.warning("The \"objectives\" option "
-                           "is deprecated for the learning_curve "
-                           "task and will not be supported "
-                           "after the next release; please "
-                           "use the \"metrics\" option in the [Output] "
-                           "section instead.")
+        if len(grid_objectives) > 0:
+            raise ValueError("The \"objectives\" option "
```

Comment:
I would rephrase this to make it a bit shorter:

"The \"objectives\" option is no longer supported for the learning_curve task. Please use the ..."

Reply:
fixed.